### PR TITLE
feat: account addresses

### PR DESCRIPTION
**Pull request by abocsan-plenty**
## Why:

Closes: #ID

## Checklist before requesting a review
- [ ] My code follows the code style of this project.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: apps/web/components/AddressForm/AddressForm.vue [hunk 1]
```diff
@@ -101,10 +101,10 @@ import type { AddressFormProps } from './types';
 const { loading: loadBilling } = useBillingAddress();
 const { loading: loadShipping } = useShippingAddress();
 const props = defineProps<AddressFormProps>();
-const isCartUpdateLoading = computed(() => loadBilling.value || loadShipping.value);
+const isCartUpdateLoading = computed(() => props.loading || loadBilling.value || loadShipping.value);
 
 const { savedAddress } = toRefs(props);
-const useAsShippingAddress = ref(true);
+const useAsShippingAddress = ref(false);
```
Comment (FabianGerke) [inline apps/web/components/AddressForm/AddressForm.vue]: Why "false" here?
We said we want that box checked ([PRODUCT BACKLOG ITEM 72842](https://dev.azure.com/plentymarkets/plentymarkets/_workitems/edit/72842))

File: apps/web/composables/useAddress/useAddress.ts
Comment (FabianGerke) [inline apps/web/composables/useAddress/useAddress.ts:1]: Maxi told me at the start of the boilerplate that he wants a composable for billing and another one for shipping.

So we currently have:
useBillingAddress
useShippingAddress

We should use them / or remove them.
@maxiroellplenty

**Review by FabianGerke — CHANGES_REQUESTED** [review]

File: apps/web/components/AddressForm/AddressForm.vue (hunk 1 above)
Comment (abocsan-plenty) [reply apps/web/components/AddressForm/AddressForm.vue]: In this task: https://dev.azure.com/plentymarkets/plentymarkets/_sprints/backlog/BD%20-%20Team%20-%20LM/plentymarkets/Areas/Sprint%2038?workitem=71438

we have:

This checkbox is unchecked every time the modal is closed / We do not save the state of it

Comment (FabianGerke) [reply apps/web/components/AddressForm/AddressForm.vue]: Than we have to implement that as a prop, that we can handle both cases.

Comment (ksted) [reply apps/web/components/AddressForm/AddressForm.vue]: From what I recall, part of the reason why we had that specification was simplicity. It's fine if the box is checked every time the modal opens I'd say.

Comment (abocsan-plenty) [reply apps/web/components/AddressForm/AddressForm.vue]: I talked with Tim, it should be different behaviors in the myaccount and checkout, I added a prop with default to true.

File: apps/web/composables/useAddress/useAddress.ts
Comment (abocsan-plenty) [reply apps/web/composables/useAddress/useAddress.ts:1]: I removed `useBillingAddress` and `useShippingAddress` and refactored checkout to use `useAddress`

**Review by FabianGerke — APPROVED** [review]

**Pull request merged**